charts/argo-cd/README.md.gotmpl

```diff
@@ -48,16 +51,16 @@ redis-ha:
   enabled: true
 
 controller:
-  enableStatefulSet: true
+  replicas: 2
```

Comment:
use 1 as default. See other comment

Reply:
1 is not actually great for HA example

Reply:
The controller simply does not support HA.
Ref: https://github.com/argoproj/argo-cd/pull/8340/files?diff=unified&w=0&short_path=5ab05fa#diff-5ab05fafb9f84e2256b1b32e91f4de69e89e5eba19eae3818c7aba539fb22fb0

> Currently, the application-controller does not provide high availability to support workload distribution. This causes conflicts and leads to orphaned/unmanaged applications in scenarios where the application controller handling these resources malfunctions/crashes.

Reply:
Good to know - I always thought it can handle failure. Anyway updated to 1